docs/source/concepts/sim_settings.md

```diff
@@ -50,3 +50,73 @@ The pre-implemented workflows hold the needed settings for the simulation with
 the pre-implemented Plugins. For further customization you can add your own 
 Workflow (from scratch or by inheriting from the existing ones) and add your own
 settings that you might miss from the existing ones.
+
+## List of current simulation settings:
+The following tables provide an overview of simulation settings for the 
+different simulation types.
+
+### BaseSimSettings
+
+| Setting Name             | Type         | Default       | Description                                                                                                       |
+|--------------------------|--------------|---------------|-------------------------------------------------------------------------------------------------------------------|
+| dymola_simulation        | Boolean      | False         | Run a Simulation with Dymola after model export?                                                                  |
+| create_external_elements | Boolean      | False         | Create external elements?                                                                                         |
+| max_wall_thickness       | Number       | 0.3           | Choose maximum wall thickness as a tolerance for mapping opening boundaries to their base surface (Wall).         |
+| group_unidentified       | Choice       | 'fuzzy'       | To reduce the number of decisions by user to identify elements which can not be identified automatically.         |
```

Comment:
shouldn't we list all available options here additionally to the default? At least for the 'Choice' parameters

Reply:
this was just a very fast implementation (thanks to ChatGPT). We can improve this later, but as it's not really part of this PR I would like to leave it like it is for now.